de/matrix.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from factorization import cellLineFactorization
+
+
+def cellLineComparision(cellLine1, cellLine2):
+    w1, eta1, annotation1 = cellLineFactorization(cellLine1)
+    w2, eta2, annotation2 = cellLineFactorization(cellLine2)
+
+    norm1 = np.linalg.norm(w1)
+    norm2 = np.linalg.norm(w2)
+
+    line1_as_set = set(annotation1)
+    intersection = line1_as_set.intersection(annotation2)
```

Comment:
This is a great way to find the intersection. You can then use these sets to find the indices in the original lists, using the link I gave in the comment.

Reply:
For finding the indices, I used the code below from lines 18-25.